### PR TITLE
Breaking long URLS to prevent overflow

### LINKED ISSUE #6329: LiveChat Links Not Wrapping as Expected
Rocket.Chat Version: 0.52.0
Running Instances: N/A
DB Replicaset OpLog: N/A
Node Version: N/A

[OpenEMR](http://open-emr.org/) is now using RocketChat (awesome work, all!) and we're noticing that long links don't wrap as expected:

image

Should be super easy to fix by following: https://css-tricks.com/snippets/css/prevent-long-urls-from-breaking-out-of-container/

Thanks,
Matthew

### PR DESCRIPTION
**Pull request by robertdown**
Added CSS to handle long URL overflow

@RocketChat/core 

Closes #6329 

Updated box
screen shot 2017-03-15 at 19 47 25

### CODE REVIEW

File: packages/rocketchat-livechat/app/client/stylesheets/main.less [hunk 1]
```diff
@@ -258,6 +258,13 @@ input:focus {
 					border-radius: 6px;
 					padding: 5px;
 					float: left;
+					word-wrap: break-word;
+					word-break: break-word;
+					-ms-word-break: break-all;
```
Comment (karlprieb) [inline packages/rocketchat-livechat/app/client/stylesheets/main.less]: I think we don't need this. IE11 can handle `word-break` already.

Comment (rodrigok) [inline packages/rocketchat-livechat/app/client/stylesheets/main.less]: @karlprieb can we enable the less autoprefixer again?

Comment (sampaiodiego) [inline packages/rocketchat-livechat/app/client/stylesheets/main.less]: @karlprieb is this the only concern on this PR? if so, we can edit it to use only `work-break` and merge it

Comment (karlprieb) [inline packages/rocketchat-livechat/app/client/stylesheets/main.less]: @sampaiodiego Of course :)

Comment (robertdown) [inline packages/rocketchat-livechat/app/client/stylesheets/main.less]: Just to clarify, you want the `-ms-word-break` removed, correct?

**Pull request merged**